pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -68,6 +68,8 @@ const prefixMatchRegex = `((\/).*)?`
 
 var notimeout = durationpb.New(0)
 
+type DestinationHashMap map[*networking.HTTPRouteDestination]*networking.LoadBalancerSettings_ConsistentHashLB
```

Comment:
Using pointer type key is very dangerous

Reply:
This is existing one only. I just introduced struct.